Controller/Payment/Notify.php

```diff
@@ -140,6 +148,13 @@ public function execute()
                 $kevinSignature = $this->getRequest()->getHeader('X-Kevin-Signature');
                 $method = $this->getRequest()->getMethod();
                 $url = $this->getRequest()->getUriString();
+                //$signature = $this->config->getSignature();
+
+                $this->logger->info('Callback Body: '.$body);
+                $this->logger->info($timestamp);
+                $this->logger->info($kevinSignature);
+                $this->logger->info($method);
+                $this->logger->info($url);
```

Comment:
We do not need it. We can lookup such things in our internal logs.

Reply:
I think it's ok, could be helpful to shop administrators.

Reply:
The only reason why the one would like to see such information is when the status does not change. Otherwise, it does not add any value in my opinion.

Reply:
That is true, it's hard to see how it could be useful in practice, better have it removed.

Reply:
Ok, I will remove this log in second commit